api/tests/unit/organisations/chargebee/test_unit_chargebee_chargebee.py

```diff
@@ -260,8 +261,9 @@ def test_get_hosted_page_url_for_subscription_upgrade(self):
         )
 
 
-@pytest.mark.parametrize("addon_quantity", (None, 1))
-def test_get_subscription_metadata(mocker, chargebee_object_metadata, addon_quantity):
+def create_mocked_subscription_metadata(
+    mocker, chargebee_object_metadata, addon_quantity
+):
```

Comment:
Can we make this a fixture that returns a callable? Also, we need to add type hints to each of the fixtures and the return type.